x-pack/platform/plugins/private/data_visualizer/public/lite/import_errors.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiCallOut, EuiSpacer } from '@elastic/eui';
+import type { FC } from 'react';
+import React from 'react';
+import { type UploadStatus } from './file_manager/file_manager';
+
+interface Props {
+  uploadStatus: UploadStatus;
+}
+
+export const ImportErrors: FC<Props> = ({ uploadStatus }) => {
+  return (
+    <>
+      <EuiSpacer />
+      {uploadStatus.errors.map((error, index) => (
+        <>
```

Comment:
you need to provide a `key` here

Reply:
Updated in https://github.com/elastic/kibana/pull/206864/commits/8bf59f8a149da53e5f73aa55cf472bf9df13c11b